examples/event_handling/data_browser.py

```diff
@@ -3,6 +3,8 @@
 Data Browser
 ============
 
+An example of how to interact with multiple canvas by selecting and highlighting
```

Comment:
I'd split this into two blocks. The first sentence should be quite short and fit one one line. Something like:

```
Connecting data between multiple canvases.

This example covers how to interact data with multiple canvases. This
let's you select and highlight a point on one axis, and generating the
data of that point on the other axis.
```